src/routes/bounce_activities_page.jsx

```diff
@@ -37,12 +38,16 @@ export default class BounceActivityPage extends React.Component {
   }
 
   async componentDidMount() {
-    const { data: activities } = await getActivityLog();
-    if (activities) {
-      this.setState({
-        isFetching: false,
-        activityLog: activities.reverse(),
-      });
+    try {
+      const { data: activities } = await getActivityLog();
+      if (activities) {
+        this.setState({
+          isFetching: false,
+          activityLog: activities.reverse(),
+        });
+      }
+    } catch (err) {
+      this.setState({ isNetworkError: true });
```

Comment:
add `isFetching: false` since when it errors out it shouldn't show the loading icon any more